.github/workflows/e2e-tests.yml

```diff
@@ -6,16 +6,19 @@ on:
   pull_request:
 jobs:
   e2e-tests:
+    strategy:
+      matrix:
+        parallel-id: [0, 1, 2, 3]
```

Comment:
Ah TIL that build matrix strategies don't play well with existing required checks defined through branch protection policies. We may need to work towards a model where we remove required checks entirely (which may implicitly imply we need to remove tide entirely for this repository).

Reply:
Yeah unfortunately actions' matrix capabilities, like required checks and rerunning failed matrix cells only, are lacking.

Reply:
I think my preference would be to temporarily remove this as required checks vs. purging tide entirely (which I'd still like to do long term). I can work towards allowlisting the e2e-tests as a optional check and we'll just have reviewers ensure they're passing before labeling any PR.